FogCreek/Glitch-Community
# West jumbo: Add Team Project popover

## Links
* Remix link: https://west-jumbo.glitch.me
* Manuscript link: https://glitch.manuscript.com/f/cases/3328661/

## GIF/Screenshots:
![Screen Shot 2019-05-29 at 10 54 01 AM](https://user-images.githubusercontent.com/938722/58567297-1f678680-8200-11e9-9f2b-d4ebb10c367d.png)

## Changes:
* rebuild Add Team Project popover with components
* remove unused "templates" search

## How To Test:
* log into https://west-jumbo.glitch.me
* create a team
* click Add Project to open the Add Team Project popover
* the project results should be filterable and keyboard navigable
* clicking on a project should add it to the team
